controllers/funding/README.md

```diff
@@ -0,0 +1,3 @@
+# Funding landing router
+
+Router for the top-level `/funding` page. This is largely a static page with a dynamic list of the latest programmes pulled in from the CMS.
```

Comment:
My PR isn't merged yet but this will shortly include the "Publications" section. I can add to these docs when this gets merged, though.